transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodPresenter.java

```diff
@@ -268,14 +284,28 @@ public void onKeysLoaded(@Nullable HyperwalletTransferMethodConfigurationKey key
                 if (!mView.isActive()) {
                     return;
                 }
-
                 Set<Currency> currencyCodes = key.getCurrencies(countryCode) != null ?
                         key.getCurrencies(countryCode) : new HashSet<Currency>();
+                // Find the country based on the country code
+                Country selectedCountry = null;
```

Comment:
I noticed this code is duplicated, Can you create method to get country from `HyperwalletTransferMethodConfigurationKey`?